test/linalg/dense.jl

```diff
@@ -512,6 +512,20 @@ end
     @test_throws ArgumentError diag(zeros(0,1),2)
 end
 
+@testset "Matrix to real power" for elty in (Float64, Complex{Float64})
+# Tests proposed at Higham, Deadman: Testing Matrix Function Algorithms Using Identities, March 2014
+    A11 = rand(elty, 10, 10)
```

Comment:
And we should test real matrices that have (a) all positive real eigenvalues, (b) a mix of positive and negative real eigenvalues and (c) complex eigenvalues.